src/Npgsql/NpgsqlDataReader.cs

```diff
@@ -1409,14 +1429,16 @@ public new NpgsqlNestedDataReader GetData(int ordinal)
     /// <returns>The actual number of bytes read.</returns>
     public override long GetBytes(int ordinal, long dataOffset, byte[]? buffer, int bufferOffset, int length)
     {
+        ThrowIfNotInResult();
+        var field = RowDescription[ordinal];
+
         if (dataOffset is < 0 or > int.MaxValue)
-            throw new ArgumentOutOfRangeException(nameof(dataOffset), dataOffset, $"dataOffset must be between {0} and {int.MaxValue}");
+            throw new ArgumentOutOfRangeException(nameof(dataOffset), dataOffset, "dataOffset must be between 0 and " + int.MaxValue);
```

Comment:
Simplifies codegen by removing the low value interpolation

Open to reverting it too